test_manager/features/steps/tck_step_implementations.py

```diff
@@ -82,11 +83,97 @@ def create_subprocess(command: List[str]) -> subprocess.Popen:
     return process
 
 
+def cast_data_to_jsonable_bytes(value: str):
+    return "BYTES:" + value
+
+def cast_data_to_bytes(value: str):
```

Comment:
Please format this file, before function definition expected 2 blank lines.

Reply:
done, formatted the file according to guidelines